packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -41,19 +61,19 @@ class MemoryController {
   }
 
   /// Listeners to hookup modifying the MemoryChartState.
-  final List<chartStateListener> _resetFeedListeners = [];
+  final List<chartStateListener> _memorySourceListeners = [];
 
-  void addResetFeedListener(chartStateListener listener) {
-    _resetFeedListeners.add(listener);
+  void addMemorySourceListener(chartStateListener listener) {
+    _memorySourceListeners.add(listener);
   }
 
-  void removeResetFeedListener(chartStateListener listener) {
-    _resetFeedListeners.remove(listener);
+  void removeMemorySourceListener(chartStateListener listener) {
+    _memorySourceListeners.remove(listener);
   }
 
-  // Call any ChartState listeners.
-  void notifyResetFeedListeners() {
-    for (var notifyListener in _resetFeedListeners) {
+  // Call any ChartState listeners that the memory source has changed.
+  void notifyMemorySourceListeners() {
+    for (var notifyListener in _memorySourceListeners) {
       notifyListener();
```

Comment:
instead of manually managing listeners, why not just use a value notifier that stores the memory source? then clients can use the ValueListenableBuilder widget to listen for data updates. There are examples of this in other pages.

Reply:
Done.